src/main/java/mediator/Mediator.java

```diff
@@ -161,7 +161,7 @@ private synchronized Carrier createCarrier(List<RfPort> rfPorts, FrequencyBand c
      * List any registered radio units that contain the parameter being passed.
      *
      * @param param The parameter that will be used to query for radio units. Examples
-     *              include a RatType, a RadioUnitState, a FrequencyBand, or a String name.
+     *              include a RatType, a RadioUnitStateE, a FrequencyBand, or a String name.

```

Comment:
To clarify, what was RadioUnitStateE? Is it the error?

Reply:
I believe it is the enum that contains the different RadioUnitStates.

Reply:
Ah, that makes sense.